config-overrides.js

```diff
@@ -13,6 +13,13 @@ module.exports = {
 
     config = _.merge(config, manualOverrides)
 
+    config.module.rules[2].oneOf.unshift(
```

Comment:
Why did you add this? It looks like you need to update from dev on your branch.

Reply:
I think I corrected the issue with the extraneous files that were added in a rebase I had done.
I am now seeing 3 files changed (as it should be). If you still see this please flag it again - thanks!